.github/workflows/release.yml

```diff
@@ -175,17 +160,17 @@ jobs:
   update_arch:
     name: Update Arch Linux Packager
     runs-on: ubuntu-latest
-    needs: [create_release, build_upload_github_assets]
-    if: needs.create_release.outputs.is_prerelease == 'false'
+    needs: [test, release]
+    if: needs.test.outputs.is_prerelease == 'false'
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Calculate New SHAs
         id: calculate-shas
         run: |
           PKG=step-cli-bin
-          VVER=${{ needs.create_release.outputs.vversion }}
-          VER=${{ needs.create_release.outputs.version }}
+          VVER=${{ needs.test.outputs.vversion }}
+          VER=${{ needs.test.outputs.version }}
```

Comment:
A few lines below we have 
```
echo "${{ secrets.ARCH_SSH_KEY }}" > /tmp/ssh_arch_key && chmod 0400 /tmp/ssh_arch_key
```
Does this key appear in the logs?

Reply:
https://docs.github.com/en/actions/reference/encrypted-secrets#accessing-your-secrets

It shouldn't. This isn't exactly recommended, but I'm open to suggestions.